pom.xml

```diff
@@ -75,7 +81,7 @@
             <dependency>
                 <groupId>commons-io</groupId>
                 <artifactId>commons-io</artifactId>
-                <version>2.7</version>
+                <version>2.11.0</version>
```

Comment:
Since you're updating the version, can you move them to a property ? This will make it easier to update versions when needed (for this dependency and others below as well)

Reply:
I'll

Reply:
done